.github/workflows/build_and_test.yml

```diff
@@ -614,3 +614,103 @@ jobs:
           docker tag rocketchat/${{ matrix.service }}-service:${IMAGE_TAG} rocketchat/${{ matrix.service }}-service:${RELEASE}
           docker push rocketchat/${{ matrix.service }}-service:${RELEASE}
         fi
+
+  release_snap_amd64:
+    env:
+      SNAPCRAFT_REPO: RocketChat/rocketchat-server-snap
+      TESTS_REPO: RocketChat/rocketchat-release-tests
+    runs-on: ubuntu-18.04
+    needs: deploy
+    if: github.event_name == 'release'
+    steps:
+      - name: Parse environment variables
+        run: |
+          export ROCKETCHAT_VERSION=${{ github.event.release.tag_name }}
+          echo "CHANNEL=${ROCKETCHAT_VERSION:0:1}.x" >> $GITHUB_ENV
```

Comment:
I’d call this SNAP_TRACK instead of CHANNEL.

Reply:
Also we need to be careful… tracks can’t be opened dynamically.  We have to explicitly open them by request.  I guess we could leave it and the push step will just fail.  But maybe we should add a comment above this with that as a note